contrib/fonttools/showttf.c

```diff
@@ -1,3 +1,6 @@
+#if 0
```

Comment:
???

Reply:
How could it have already been there when the diff shows it as being added?

Reply:
merged from here https://packages.debian.org/sid/fontforge-extras

Reply:
So we took an older version of the source ([1](https://github.com/fontforge/fontforge/blob/5aa251a3230006ce20d68ba533fc59dff16c6c79/contrib/fonttools/showttf.c), [2](http://metadata.ftp-master.debian.org/changelogs/main/f/fontforge-extras/fontforge-extras_0.3-4_changelog)) and copy pasted it back in?

Either way, I don't think it's great to be adding in redundant code.